WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -20,45 +19,35 @@ def new
   end
 
   def index
-    # We use the ["",""] to separate the items. They are disabled in the view, so the
-    # user cannot select an empty param.
-    @regions = { 'Continent' => Continent.all.map { |continent| [continent.name, continent.id] },
-                 'Country' => Country.all.map { |country| [country.name, country.id] } }
-    @years = [ ["Current","current"],["All","all"],["",""] ] + Competition.where(showAtAll: true).map(&:year).uniq.sort.reverse!
-    @competitions = Competition.where(showAtAll: true).order(:year, :month, :day).reverse_order
-
-    # This needs to be the first thing, otherwise @competitions will be an array instead of an object
-    # and the .where will not work.
-    if params[:years].blank?
-      params[:years] = "current"
-    end
-
-    if params[:region].blank?
-      params[:region] = "all"
-    end
-
-    if params[:display].blank?
-      params[:display] = "List"
-    end
-
+    # Default params
     params[:event_ids] ||= []
+    params[:region] ||= "all"
+    params[:state] ||= "present"
+    params[:year] ||= "all years"
+    params[:display] ||= "list"
 
     if params[:event_ids].is_a?(Hash)
```

Comment:
I'm already forgetting what this is about, could we add a comment like `Workaround for https://github.com/cubing/worldcubeassociation.org/issues/472`?